### PR TITLE
[T] (quests) botania

### PR DESCRIPTION
**Pull request by nai200**
Co-authored-by: 墨羽千寻 <169262849@qq.com>

### CODE REVIEW

File: config/ftbquests/quests/chapters/botania.snbt [hunk 1]
```diff
@@ -2676,15 +2677,15 @@
 			x: 7.75d
 			y: 8.75d
 			shape: "diamond"
-			subtitle: "The Orechid is a great flower"
+			subtitle: "炎矿兰是一朵很好的花"
 			description: [
-				"The ability to create raw ores from &aMana&r and &aStone&r can be a great boon to any sedentary botanist's life."
+				"从&aMana&r和&a石头&r中创造原矿的能力对任何植物学家来说都是一个巨大的福音。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"用&a魔力&r和&a石头&r创造原矿的能力对任何植物学家来说都是一个巨大的福音。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 2]
```diff
@@ -2623,8 +2624,8 @@
 			x: 2.5d
 			y: 8.25d
 			shape: "diamond"
-			subtitle: "The Endoflame is a very rudimentary generating flower"
-			description: ["it'll absorb any combustible items or blocks dropped on the nearby vicinity, one at a time, and burn through them to generate &aMana&r."]
+			subtitle: "火红莲是一种非常初级的产能花"
+			description: ["它会吸收掉在附近的任何可燃物品或方块，一次一个，并燃烧它们产生&aMana&r。"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
			description: ["它会吸收掉在附近的任何可燃物品或方块，一次一个，并燃烧它们产生&a魔力&r。"]
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 3]
```diff
@@ -2567,11 +2568,11 @@
...
 			y: 9.0d
 			shape: "diamond"
-			subtitle: "To generate a blast of Mana, in fact, a blast might be required."
+			subtitle: "事实上，为了产生一波魔力，就需要一波爆炸。"
 			description: [
-				"Igniting a block of &dTNT&r on dry land near an &dEntropinnyum&r will have the latter absorb all the entropy generated by the blast, converting it into &aMana&r and nullifying any damage in the process."
+				"靠近热爆花点燃一个&dTNT&r，&d热爆花&r就会吸收爆炸产生的热和冲击并产生大量的&aMana&r消除一切爆炸伤害。"
 				""
-				"However, to absorb the explosion, the flower must not have any &aMana&r stored in it-- otherwise, explosions as usual."
+				"最后，为了吸收爆炸，花朵本身需要无&aMana&r，否则吸收将会失败。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"最后，为了吸收爆炸，花朵本身需要无&a魔力&r，否则吸收将会失败。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 4]
```diff
@@ -2567,11 +2568,11 @@
 			x: 1.0d
 			y: 9.0d
 			shape: "diamond"
-			subtitle: "To generate a blast of Mana, in fact, a blast might be required."
+			subtitle: "事实上，为了产生一波魔力，就需要一波爆炸。"
 			description: [
-				"Igniting a block of &dTNT&r on dry land near an &dEntropinnyum&r will have the latter absorb all the entropy generated by the blast, converting it into &aMana&r and nullifying any damage in the process."
+				"靠近热爆花点燃一个&dTNT&r，&d热爆花&r就会吸收爆炸产生的热和冲击并产生大量的&aMana&r消除一切爆炸伤害。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"靠近热爆花点燃一个&dTNT&r，&d热爆花&r就会吸收爆炸产生的热和冲击并产生大量的&a魔力&r消除一切爆炸伤害。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 5]
```diff
@@ -2539,11 +2540,11 @@
...
 			y: 9.75d
 			shape: "diamond"
-			subtitle: "The Gourmaryllis is one hungry flower."
+			subtitle: "彼方兰是一种饥饿的花。"
 			description: [
-				"Thus, a &dSteak&r will take four seconds to digest, an &dApple&r will take two, a loaf of &dBread&r will take two and a half, and so on."
+				"因此，一份&d牛排&r需要四秒钟消化，一个&d苹果&r需要两秒钟消化，一条&d面包&r需要两秒半消化，以此类推。"
 				""
-				"The rate of &aMana&r production will also vary with nutrition: the &dGourmaryllis&r prefers bigger foods."
+				"&aMana&r的产量也会因营养而异，&d彼方兰&r更喜欢保持度多的食物。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"&a魔力&r的产量也会因营养而异，&d彼方兰&r更喜欢保持度多的食物。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 6]
```diff
@@ -2515,8 +2516,8 @@
 			x: 3.5d
 			y: 9.75d
 			shape: "diamond"
-			subtitle: "Cake is delicious; everyone loves it, flowers included."
-			description: ["The &dKekimurus&r is one of these &dCake&r aficionados, and will eat any placed in its vicinity, synthesizing the enriching nutrients into sweet, sweet &aMana&r."]
+			subtitle: "蛋糕很好吃；每个人都喜欢它，包括花儿。"
+			description: ["&d贪食花&r就是这些&d蛋糕&r的狂热爱好者之一，他们会吃任何放置在它附近的蛋糕，将丰富的营养合成成甜甜的&aMana&r。"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
			description: ["&d贪食花&r就是这些&d蛋糕&r的狂热爱好者之一，他们会吃任何放置在它附近的蛋糕，将丰富的营养合成成甜甜的&a魔力&r。"]
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 7]
```diff
@@ -2462,11 +2463,11 @@
...
 			y: 9.0d
 			shape: "diamond"
-			subtitle: "There's some unidentified power in the universe that creates Slimes at certain points, seemingly at random."
+			subtitle: "世界上还存在着某种未知的能量场，那里会在特定的时间点上，似乎是随机地创造出史莱姆。"
 			description: [
-				"Slime Chunks, they call those points. The &dNarslimmus&r is a flower that has the ability to tap into that energy; it'll absorb &aSlimes&r created by that power and collect all the Mana generated by the destruction."
+				"人们称那些场为史莱姆区块。&d粘球草&r就能利用这种能量。如果它被种在了由这种能量创造出来的&a史莱姆&r旁边，它就会分解史莱姆，并收集这种能量，并且产生魔力。"
 				""
-				"The larger the &aSlime,&r the more &aMana&r is created. (It also makes a bit of a sticky mess.)"
+				"&a史莱姆&r越大，其产生的&aMana&r越多。（这也造成了一些黏乎乎的麻烦。）"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"&a史莱姆&r越大，其产生的&a魔力&r越多。（这也造成了一些黏乎乎的麻烦。）"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 8]
```diff
@@ -2437,8 +2438,8 @@
 			x: 1.5d
 			y: 9.75d
 			shape: "diamond"
-			subtitle: "The Rafflowsia functions similarly to a Kekimurus"
-			description: ["It'll consume any nearby placed flowers and synthesize &aMana&r from them. While feeding it the same flower several times in a row yields diminishing returns, feeding it a large variety of them can yield ludicrous quantities of &aMana&r."]
+			subtitle: "噬草花的功能类似于贪食花"
+			description: ["它会寻找附近放置的所有花朵并消耗掉，并以此产生&aMana&r。连续几次投喂同一种花的收益递减，而投喂大量不同种类的花可以产生大量的&aMana&r。"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
			description: ["它会寻找附近放置的所有花朵并消耗掉，并以此产生&a魔力&r。连续几次投喂同一种花的收益递减，而投喂大量不同种类的花可以产生大量的&a魔力&r。"]
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 9]
```diff
@@ -2412,8 +2413,8 @@
 			x: 1.0d
 			y: 10.5d
 			shape: "diamond"
-			subtitle: "The Spectrolus is a flower that's particularly fond of the various hues of Wool."
-			description: ["It'll consume any and all &dWool&r blocks dropped nearby, converting them to &aMana&r. However, it's picky as to what colors it wants. Starting from &bWhite&r, after it eats one piece of &bWool&r, it'll rotate to the next color in the spectrum."]
+			subtitle: "斑斓花是一种特别喜欢各种颜色的羊毛的花。"
+			description: ["它将吸收任何掉落在它附近的&d羊毛&r，将它们转换为&aMana&r。然而，它对颜色十分挑剔。从&b白色&r开始，吸收一块&b羊毛&r后，它会按照顺序所求下一个颜色的羊毛。"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
			description: ["它将吸收任何掉落在它附近的&d羊毛&r，将它们转换为&a魔力&r。然而，它对颜色十分挑剔。从&b白色&r开始，吸收一块&b羊毛&r后，它会按照顺序所求下一个颜色的羊毛。"]
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 10]
```diff
@@ -2383,11 +2384,11 @@
 			x: 9.75d
 			y: 8.75d
 			shape: "diamond"
-			subtitle: "While going mining is well and good, a renewable and sedentary means of acquiring ores is nothing to scoff at either."
+			subtitle: "尽管采矿很不错，但我们应该想些可再生的获取矿石的方式。"
 			description: [
-				"The Orechid uses &aMana&r to synthesize &aores&r from nearby &bStone&r blocks."
+				"凝矿兰使用&aMana&r将附近的&b石头&r块转化为&a矿物&r。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"凝矿兰消耗&a魔力&r将附近的&b石头&r块转化为&a矿物&r。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 11]
```diff
@@ -2356,11 +2357,11 @@
...
 			y: 8.75d
 			shape: "diamond"
-			subtitle: "The Clayconia is a simple flower."
+			subtitle: "黏土花是一种简单的花。"
 			description: [
-				"All it does is moisten nearby &dSand&r, turning it into pellets of &dClay&r."
+				"它所做的就是润湿附近的&d沙子&r，把它变成&d黏土&r颗粒。"
 				""
-				"This process uses a decent amount of &aMana&r, but won't break the bank (or pool, as it were)."
+				"这个过程使用了相当数量的&aMana&r，但不会耗尽整个库存（或池）。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"这个过程使用了相当数量的&a魔力&r，但不会耗尽整个库存（或池）。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 12]
```diff
@@ -2327,11 +2328,11 @@
 			x: 10.25d
 			y: 9.5d
 			shape: "diamond"
-			subtitle: "Any adventurer knows that Dungeons can hold valuable goodies."
+			subtitle: "任何冒险家都知道地下城可以储存有价值的东西。"
 			description: [
-				"The &dLoonium&r will, when fed quite a bit of &aMana&r, summon these items for said adventurers to collect. There's a catch, though: each item is held by a monster protecting it."
+				"当被喂食相当多的&aMana&r时，&d聚宝花&r可以使地牢里的奖励成为可收集的物品。不过，有一个陷阱。除了这个物品外，还有一个怪物被用来保护它。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"当被喂食相当多的&a魔力&r时，&d聚宝花&r可以使地牢里的奖励成为可收集的物品。不过，有一个陷阱。除了这个物品外，还有一个怪物被用来保护它。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 13]
```diff
@@ -2304,8 +2305,8 @@
 			x: 3.0d
 			y: 10.5d
 			shape: "diamond"
-			subtitle: "Generating the form of natural energy that is Mana is best done from other living materials."
-			description: ["The &dMunchdew&r will eat up any nearby &dLeaves&r and convert them into &aMana&r. However, no saplings (or any other items) will be dropped from the leaves."]
+			subtitle: "产生自然能量，即Mana，的最佳方式是从其他生物材料中提取。"
+			description: ["&d咀叶花&r将吃掉附近的任何&d树叶&r，并将它们转化为&aMana&r。但是，不会有树苗（或任何其他物品）从叶子里掉出来。"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
			description: ["&d咀叶花&r将吃掉附近的任何&d树叶&r，并将它们转化为&a魔力&r。但是，不会有树苗（或任何其他物品）从叶子里掉出来。"]
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 14]
```diff
@@ -2304,8 +2305,8 @@
 			x: 3.0d
 			y: 10.5d
 			shape: "diamond"
-			subtitle: "Generating the form of natural energy that is Mana is best done from other living materials."
-			description: ["The &dMunchdew&r will eat up any nearby &dLeaves&r and convert them into &aMana&r. However, no saplings (or any other items) will be dropped from the leaves."]
+			subtitle: "产生自然能量，即Mana，的最佳方式是从其他生物材料中提取。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
			subtitle: "产生自然能量，即魔力，的最佳方式是从其他生物材料中提取。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 15]
```diff
@@ -2278,13 +2279,13 @@
 			x: 8.25d
 			y: 9.5d
 			shape: "diamond"
-			subtitle: "The Hopperhock simply picks up items around it and put them in adjacent inventories."
+			subtitle: "漏斗花只是机械地拾取周围的物品，并将它们放入相邻的存储容器中。"
 			description: [
-				"&aMana&r is optional, but will increase its range if provided. Placing &dItem Frames&r on adjacent inventories will specify which items can or can't go in them."
+				"&aMana&r不是必要的，但如果提供将增加其范围。将&d物品展示框&r放置在相邻的存储容器中将指定哪些项目可以或不可以放入其中。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"&aM魔力&r不是必要的，但如果提供将增加其范围。将&d物品展示框&r放置在相邻的存储容器中将指定哪些物品可以或不可以放入其中。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 16]
```diff
@@ -2254,11 +2255,11 @@
 			x: 9.25d
 			y: 9.5d
 			shape: "diamond"
-			subtitle: "Find it in a sandpit?"
+			subtitle: "在沙坑里找到的?"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
			subtitle: "在沙坑里找到的？"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 17]
```diff
@@ -2234,8 +2235,8 @@
 			x: 7.25d
 			y: 9.5d
 			shape: "diamond"
-			subtitle: "The slow growth of crops is a perpetual problem in the feeding of the masses"
-			description: ["The &dAgricarnation&r transforms &aMana&r into a type of natural fertilizer, causing nearby plant-life to grow faster."]
+			subtitle: "人民群众吃饭的一个永恒的问题是庄稼长得太慢了"
+			description: ["田园康乃馨将&aMana&r转化为一种天然肥料，使附近的植物生长得更快。"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
			description: ["田园康乃馨将&a魔力&r转化为一种天然肥料，使附近的植物生长得更快。"]
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 18]
```diff
@@ -2210,11 +2211,11 @@
 			x: 1.5d
 			y: 8.25d
 			shape: "diamond"
-			subtitle: "Hydroangeas flowers act as liquid-based passive generators."
+			subtitle: "水绣球就像被动液体发电机。"
 			description: [
-				"They suck up any &dstill water&r in a 3x3 area (at the same altitude) around them, converting the water into &aMana&r."
+				"它们会吸收周围3x3区域内的&d静止的水&r（在同一高度），并将水转化为&aMana&r。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"它们会吸收周围3x3区域内的&d静止的水&r（在同一高度），并将水转化为&a魔力&r。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 19]
```diff
@@ -2210,11 +2211,11 @@
...
 				""
-				"Unfortunately, even though they seem to function faster during &aRain&r, their base &aMana&r throughput is still rather slow. In addition, they decay after around three days, making them nonviable for long-term Mana production."
+				"不幸的是，尽管他们在&a雨天&r期间似乎运行得更快，但他们的基础&aMana&r吞吐量仍然相当少。此外，他们会在大约3天后腐烂，使他们无法长期生产魔力。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"不幸的是，尽管他们在&a雨天&r期间似乎运行得更快，但他们的基础&a魔力&r吞吐量仍然相当少。此外，他们会在大约3天后腐烂，使他们无法长期生产魔力。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 20]
```diff
@@ -2093,27 +2094,27 @@
 			x: -2.75d
 			y: -3.0d
 			description: [
-				"The &aCorporea&r system uses the energy of the &aEnd&r, and the void between worlds, to transport items from point A to point B instantly without intervening blocks.The system, at its core, uses variant &5Sparks&r (called &5Corporea Sparks&r) that transport items instead of &aMana&r."
+				"&a多媒体&r系统是一个利用&a末地&r的能量和世界的空白空间，将物品从A点瞬间到B点，无需任何连接部件。这个系统通过&5火花&r的变种来运作（称为&5多媒体火花&r），它们运输物品而不是&aMana&r。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"&a多媒体&r系统是一个利用&a末地&r的能量和世界的空白空间，将物品从A点瞬间到B点，无需任何连接部件。这个系统通过&5火花&r的变种来运作（称为&5多媒体火花&r），它们运输物品而不是&a魔力&r。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 21]
```diff
@@ -1317,14 +1318,14 @@
 			x: 5.75d
 			y: -6.0d
 			description: [
-				"Using &5Mana Spreaders&r to transport &aMana&r is all well and good, but there are other ways of getting the stuff around too."
+				"使用&5魔力发射器&r来传输&aMana&r是不错的选择，但也有其他方法做到这件事。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"使用&5魔力发射器&r来传输&a物品&r是不错的选择，但也有其他方法做到这件事。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 22]
```diff
@@ -1317,14 +1318,14 @@
...
 				""
-				"The &5Mana Tablet&r is a portable item that can carry &aMana&r within itself."
-				"In addition, other items in an inventory can draw from it for their own use, making it an essential tool."
+				"&5魔力石板&r是个重要的物品，用来存储&aMana&r到它自己里。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"&5魔力石板&r是个重要的物品，用来存储&a物品&r到它自己里。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 23]
```diff
@@ -1317,14 +1318,14 @@
...
+				"此外，其他消耗魔力的工具会从它里面提取魔力，使其成为必不可少的工具。"
 				""
-				"Tossing a tablet into a &5Mana Pool&r will allow for flow of &aMana&r between them."
+				"将一个石板扔进一个&5魔力池&r，将允许在他们之间流动&aMana&r。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"将一个石板扔进一个&5魔力池&r，将允许&a魔力&r在他们之间流动。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 24]
```diff
@@ -1169,13 +1170,13 @@
...
 			y: -8.75d
 			description: [
-				"The &5Mana Spreader&r is the single most important tool a botanist needs to manipulate &aMana&r."
+				"&5魔力发射器&r是植物学家需要的最重要的用来传输&aMana&r的工具。"
 				""
-				"This block shoots &aMana&r from point A to point B. It faces a cardinal direction when placed, but sneak-right clicking it with a &5Wand of the Forest&r on a face points it elsewhere. It can be aimed at other blocks with a wand in &aBind Mode&r."
+				"这个方块将&aMana&r从A点射到b点。当放置时，它面对一个基本的方向，但是用一个&5森林法杖&r在一个面上右键点击它，将它指向其他地方。可以用在&a绑定模式&r的森林法杖将它瞄准向其他方块。"
 				""
-				"The &5Mana Spreader&r has a small internal buffer of &aMana&r, which will get filled by &aGenerating Flora&r bound to it. (Flowers, when placed, auto-bind to the nearest spreader.)"
+				"&5魔力发射器&r有一个小的内部&aMana&r缓冲，会被绑定它的&a产能花&r填充。（当花放置时，自动绑定到最近的发射器。）"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"&5魔力发射器&r有一个小的内部&a魔力&r缓冲，会被绑定它的&a产能花&r填充。（当花放置时，自动绑定到最近的发射器。）"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 25]
```diff
@@ -1169,13 +1170,13 @@
 			x: 0.25d
 			y: -8.75d
 			description: [
-				"The &5Mana Spreader&r is the single most important tool a botanist needs to manipulate &aMana&r."
+				"&5魔力发射器&r是植物学家需要的最重要的用来传输&aMana&r的工具。"
 				""
-				"This block shoots &aMana&r from point A to point B. It faces a cardinal direction when placed, but sneak-right clicking it with a &5Wand of the Forest&r on a face points it elsewhere. It can be aimed at other blocks with a wand in &aBind Mode&r."
+				"这个方块将&aMana&r从A点射到b点。当放置时，它面对一个基本的方向，但是用一个&5森林法杖&r在一个面上右键点击它，将它指向其他地方。可以用在&a绑定模式&r的森林法杖将它瞄准向其他方块。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"这个方块将&a魔力&r从A点射到b点。当放置时，它面对一个基本的方向，但是用一个&5森林法杖&r在一个面上右键点击它，将它指向其他地方。可以用在&a绑定模式&r的森林法杖将它瞄准向其他方块。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 26]
```diff
@@ -1169,13 +1170,13 @@
 			x: 0.25d
 			y: -8.75d
 			description: [
-				"The &5Mana Spreader&r is the single most important tool a botanist needs to manipulate &aMana&r."
+				"&5魔力发射器&r是植物学家需要的最重要的用来传输&aMana&r的工具。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"&5魔力发射器&r是植物学家需要的最重要的用来传输&a魔力&r的工具。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 27]
```diff
@@ -365,9 +365,9 @@
 			x: 1.75d
 			y: -0.25d
 			description: [
-				"The &5Lexica Botania&r is the repository of all knowledge for all botanical matters."
+				"&5植物魔法词典&r是所有植物学知识的宝库。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"&5植物魔法辞典&r是所有植物学知识的宝库。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 28]
```diff
@@ -365,9 +365,9 @@
...
 				""
-				"All well-known botanical knowledge is stored within these pages. For convenience, if a block happens to have an entry in here, it can be sneak-right clicked on with the &5Lexica Botania&r to open said entry."
+				"所有著名的植物学知识都储存在这些页面中。为了方便起见，如果一个方块恰好在里面有一个条目，可以拿着&5植物魔法词典&r按住潜行键，右键点击它来打开该条目。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"所有著名的植物学知识都储存在这些页面中。为了方便起见，如果一个方块恰好在里面有一个条目，可以拿着&5植物魔法辞典&r按住潜行键，右键点击它来打开该条目。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 29]
```diff
@@ -336,11 +336,11 @@
 			}]
 		}
 		{
-			title: "Black Lotus"
+			title: "黑莲花"
 			icon: "botania:black_lotus"
 			x: 0.25d
 			y: 1.25d
-			description: ["An elusive flower by the name of the &5Black Lotus&r exists; however, it is not known to grow or reproduce. There are no known sources of the Lotus at this time.It is known, however, that each bloom contains a good deal of concentrated &aMana&r that can be released by dissolving it inside a non-empty &5Mana Pool&r. Just throwing it in will do."]
+			description: ["有一种难以发现的花，名叫&5黑莲花&r； 然而，它不能生长或繁殖。目前还没有这种莲花的已知来源。但是，我们知道，每一朵花都含有大量的浓缩的&aMana&r可以通过溶解在一个非空的&5魔力池&r中释放出来。把它扔进去就行了。"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
			description: ["有一种难以发现的花，名叫&5黑莲花&r； 然而，它不能生长或繁殖。目前还没有这种莲花的已知来源。但是，我们知道，每一朵花都含有大量的浓缩的&a魔力&r，可以通过溶解在一个非空的&5魔力池&r中释放出来。把它扔进去就行了。"]
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 30]
```diff
@@ -38,18 +38,18 @@
...
 		}
 		{
-			title: "Spark Augment"
+			title: "火花升级"
 			icon: "botania:spark_upgrade_isolated"
 			x: -3.75d
 			y: -0.25d
 			description: [
-				"The &5Dispersive Augment&r allows a &5Spark&r to drain the &aMana&r in its pool to charge nearby players' &aMana&r-containing items."
+				"&5弥散升级&r允许&5火花&r耗尽魔力池里的&aMana&r给附近玩家的&aMana&r储存工具充能。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"&5弥散升级&r允许&5火花&r耗尽魔力池里的&a魔力&r给附近玩家的&a魔力&r储存工具充能。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 31]
```diff
@@ -38,18 +38,18 @@
...
 				""
-				"The &5Dominant Augment&r makes a &5Spark&r pull &aMana&r from nearby non-augmented &5Sparks'&r pools into its own, until the latter is full."
+				"&5聚集升级&r允许&5火花&r从附近的装有未升级的或装有弥散升级的&5火花&r的魔力池汲取&aMana&r到本身的魔力池中。直到本身的魔力池变满。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"&5聚集升级&r允许&5火花&r从附近的装有未升级的或装有弥散升级的&5火花&r的魔力池汲取&a魔力&r到本身的魔力池中。直到本身的魔力池变满。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 32]
```diff
@@ -38,18 +38,18 @@
...
 				""
-				"The &5Recessive Augment&r makes a &5Spark&r distribute all of the &aMana&r in its pool into any nearby non-augmented or Dispersive &5Sparks'&r pools, until its own is completely empty."
+				"&5扩散升级&r允许&5火花&r分发&aMana&r到装有未升级或装有弥散升级的&5火花&r的魔力池之中。直到本身的魔力池完全变空。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"&5扩散升级&r允许&5火花&r分发&a魔力&r到装有未升级或装有弥散升级的&5火花&r的魔力池之中。直到本身的魔力池完全变空。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 33]
```diff
@@ -38,18 +38,18 @@
...
 				""
-				"Finally, the &5Isolated Augment&r prevents a &5Spark&r from interacting with any Dominant or Recessive &5Sparks&r, but still allows it to interact with blocks that can receive their &aMana&r from the &5Spark&r network."
+				"最后，&5隔绝升级&r阻止&5火花&r与带有扩散或者聚集升级的&5火花&r魔力池交互魔力，但是仍然允许它和可以从&5火花&r网络接收&aMana&r的方块交互。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"最后，&5隔绝升级&r阻止&5火花&r与带有扩散或者聚集升级的&5火花&r魔力池交互魔力，但是仍然允许它和可以从&5火花&r网络接收&a魔力&r的方块交互。"
```

File: config/ftbquests/quests/chapters/botania.snbt [hunk 34]
```diff
@@ -18,11 +18,11 @@
 			x: -2.75d
 			y: -0.25d
 			description: [
-				"&5Sparks&r are interesting things. Created from a combination of raw energy and &aMana&r, they can be placed only on a few specific &aMana&r-containing or -accepting blocks (notably &5Mana Pools&r)."
+				"&5火花&r是有趣的东西啊。用原始的能量和&aMana&r制造，它们只能被放置在一些特定的能够存储&aMana&r或者接收&aMana&r的方块上（例如&5魔力池&r）。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"&5火花&r是有趣的东西啊。用原始的能量和&a魔力&r制造，它们只能被放置在一些特定的能够存储&a魔力&r或者接收&a魔力&r的方块上（例如&5魔力池&r）。"
```

**Review by Cactusstudent — CHANGES_REQUESTED** [review]
emm，量挺大

File: config/ftbquests/quests/chapters/botania.snbt [hunk 35]
```diff
@@ -2676,15 +2677,15 @@
 			x: 7.75d
 			y: 8.75d
 			shape: "diamond"
-			subtitle: "The Orechid is a great flower"
+			subtitle: "炎矿兰是一朵很好的花"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
			subtitle: "凝矿兰是一朵很好的花"
```

**Review by Ricky-fight — APPROVED** [review]

**Review by Ricky-fight — APPROVED** [review]

File: config/ftbquests/quests/chapters/botania.snbt [hunk 36]
```diff
@@ -1035,14 +1035,15 @@
 			}]
 		}
 		{
+			title: "活石台阶"
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/botania.snbt]: 为什么多了个标题

File: config/ftbquests/quests/chapters/botania.snbt [hunk 37]
```diff
@@ -1281,8 +1282,8 @@
 		{
 			x: 6.5d
 			y: -3.75d
-			subtitle: "It's a tiny potato! It believes in you!"
-			description: ["No. Really, that's it. A &5Potato&r, thrown into a &5Mana Pool&r, gains a little spark of life and joy. The &5Tiny Potato&r is lively, but patting it with a right-click will make it even more so!A Potato can also be named in an &5Anvil&r."]
+			subtitle: "它是个小土豆！它信任着你！"
+			description: ["不。真的，就是这样。一个&5土豆&r，扔进&5魔力池&r，迸发出生命与愉悦的火花&5小土豆&r是活着的，但用右键点击它会使它更加活泼!土豆也可以命名为&5Anvil&r。"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/botania.snbt]: Anvil

File: config/ftbquests/quests/chapters/botania.snbt [hunk 38]
```diff
@@ -1317,14 +1318,14 @@
...
-				"The &5Mana Tablet&r is a portable item that can carry &aMana&r within itself."
-				"In addition, other items in an inventory can draw from it for their own use, making it an essential tool."
+				"&5魔力石板&r是个重要的物品，用来存储&a物品&r到它自己里。"
+				"此外，其他消耗魔力的工具会从它里面提取魔力，使其成为必不可少的工具。"
 				""
-				"Tossing a tablet into a &5Mana Pool&r will allow for flow of &aMana&r between them."
+				"将一个石板扔进一个&5魔力池&r，将允许&a魔力&r在他们之间流动。"
 				""
-				"Sneak-right clicking (or using a &5Dispenser&r) on the &5Mana Pool&r with a &5Wand of the Forest&r will change which way the &aMana&r will flow."
+				"用&5森林法杖&r在&5魔力池&r上用鼠标右键点击（或者使用&5发射器&r）将会改变&aMana&r的流向。"
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
				"用&5森林法杖&r在&5魔力池&r上用鼠标右键点击（或者使用&5发射器&r）将会改变&a魔力&r的流向。"
```

**Review by AnzhiZhang — APPROVED** [review]

File: config/ftbquests/quests/chapters/botania.snbt (hunk 36 above)
Comment (AnzhiZhang) [reply config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
```

File: config/ftbquests/quests/chapters/botania.snbt (hunk 37 above)
Comment (AnzhiZhang) [reply config/ftbquests/quests/chapters/botania.snbt]:
```suggestion
			description: ["不。真的，就是这样。一个&5土豆&r，扔进&5魔力池&r，迸发出生命与愉悦的火花。&5小土豆&r是活着的，但用右键点击它会使它更加活泼！土豆也可以在&5铁砧&r中重命名。"]
```

**Review by AnzhiZhang — APPROVED** [review]

**Pull request merged**